main.tf

```diff
@@ -75,17 +75,16 @@ resource "azurerm_application_gateway" "ag" {
 
   dynamic "probe" {
     for_each = [for app in local.gateways[count.index].app_configuration : {
-      name          = "${app.product}-${app.component}"
-      path          = lookup(app, "health_path_override", "/health/liveness")
-      host_name     = join(".", [lookup(app, "host_name_prefix", "${app.product}-${app.component}-${var.env}"), local.gateways[count.index].gateway_configuration.host_name_suffix])
```

Comment:
aren't you reverting this PR in this change? https://github.com/hmcts/azure-platform-terraform/pull/667 

blocker